spec/numberformat.html

```diff
@@ -328,6 +328,25 @@
         1. Return _m_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-unwrapnumberformat" aoid="UnwrapNumberFormat">
+      <h1>UnwrapNumberFormat(nf)</h1>
+      <p>
+        The UnwrapNumberFormat abstract operation gets the underlying NumberFormat operation
+        for various methods which implement ECMA-402 v1 semantics for supporting initializing
+        existing Intl objects.
+      </p>
+      <emu-alg>
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot,
+          1. If ? InstanceofOperator(_nf_, %NumberFormat%),
```

Comment:
Ah!  Till explained how I was being stupid in misreading this elsewhere.  Carry on, this change looks good!